hops/serving.py

```diff
@@ -160,22 +158,23 @@ def _start_or_stop_serving_rest(serving_id, action):
                                                             response.reason, error_code, error_msg, user_msg))
 
 
-def create_or_update(artifact_path, serving_name, serving_type="TENSORFLOW", model_version=1,
+def create_or_update(serving_name, artifact_path, model_version=1, model_server=None, kfserving=False,
                              batching_enabled = False, topic_name="CREATE",  num_partitions = 1, num_replicas = 1,
-                             instances = 1, kfserving=False):
+                             instances = 1):
     """
     Creates a serving in Hopsworks if it does not exist, otherwise update the existing one.
 
     Example use-case:
 
     >>> from hops import serving
-    >>> serving.create_or_update("/Models/mnist", "mnist", "TENSORFLOW", 1)
+    >>> serving.create_or_update("mnist", "/Models/mnist", 1)
 
     Args:
-        :artifact_path: path to the artifact to serve (tf model dir or sklearn script)
         :serving_name: name of the serving to create
-        :serving_type: type of the serving, e.g "TENSORFLOW" or "SKLEARN"
+        :artifact_path: path to the artifact to serve (tf model dir or sklearn script)
```

Comment:
Instead of sklearn script, it should be flask script right?

Reply:
I'd better say python script since there's no trace of Flask in the script provided by the user. Or python script implementing the Predict class.
Done.